tests/test_rpc.c

```diff
@@ -20,6 +20,10 @@
  * limitations under the License.
  */
 
+#include <libnetconf2/messages_client.h>
+#include <libnetconf2/netconf.h>
+#include <libyang/printer_data.h>
+#include <libyang/tree_data.h>
```

Comment:
Never put headers before feature macros (`define X`), it can lead to some unexpected problems (when the headers you are including are actually recursively including the system header you defined the feature macro for so they are compiled without the macro defined and you get errors even though you have defined the feature macro).